pkg/internal/approver/manager/review.go

```diff
@@ -125,6 +125,7 @@ func (m *mngr) Review(ctx context.Context, cr *cmapi.CertificateRequest) (manage
 		)
 
 		for _, evaluator := range m.evaluators {
+			// #nosec G601 -- False positive. The function does not keep this pointer past it's scope scope.
```

Comment:
suggestion: Couple of typos still - sorry to be a pain, I know it's just a comment

```suggestion
			// #nosec G601 -- False positive. The function does not keep this pointer past its scope.
```

Reply:
It's my mistake, should have been a bit more diligent here.